solver/src/settlement_submission/flashbots_api.rs

```diff
@@ -8,6 +10,20 @@ pub struct FlashbotsApi {
     client: Client,
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct Eip1559 {
+    max_fee_per_gas: String,
+    max_priority_fee_per_gas: String,
+}
```

Comment:
Would be nice to replace the manual parsing later with anootations here. We have similar code in crate `model` where we parse U256 using `#[serde(with = "u256_decimal")]`. I don't mind too much because this is a private type that we immediately convert into something else so not needed for this PR but wanted to mention that this is usually nicer.

Reply:
With this, I have to write a custom deserializator for these structures right?

Reply:
No, we have existing `serde(with = ...)` modules that you can use.

Reply:
Our existing U256 deserialization works on decimal strings but here we have hex so they won't fit. I just mentioned to have a code example but I don't know whether the actual type should be U256 or u64 here.

Reply:
Oh yeah, brain fart. However, note that `serde_json::from_str::<U256>("\"0x42\"").is_ok()` is the default behaviour for `U256` (de)serialization. So it should "just work".

Reply:
It "just work"-ed :) fixed to plain U256